packages/host/src/Host.tsx

```diff
@@ -19,6 +20,11 @@ import AppDrawer from './AppDrawer';
 import AppBar from './AppBar';
 import Viewport from './Viewport';
 
+const Content = styled(Box)({
```

Comment:
the table was bigger than the page, but you couldn't scroll down and see the rest of it! added the overflow to allow the scroll within this container